### ISSUE TITLE
[Bug] SDDM don't work on Hyper-v VM (2nd generation)

### ISSUE THREAD

**Issue by Yivan** [sddm/sddm#1201]
Hello,

I would like to report a problem with sddm.
sddm don't work on hyper-v. The service is unable to display the sddm gui interface.
Running startx log directly in the desktop environment (lxqt).

How to reproduce, on Windows 10 Pro uptodate stable channel, Hyper-V activated.
- install fresh debian (i made with vagrant, this image: https://app.vagrantup.com/generic/boxes/debian10)
- log into the VM as root and:
`apt update && apt install -y lxqt-core xorg sddm`
- reboot the VM, it will stay in console mode.

Running `sddm` directly bring black screen, runnnig `startx` or `xinit` brings directly on lxqt desktop.
It seems sddm service don't start, but if enabled/started manually (and rerunning `dpkg-reconfigure sddm` and calling sddm from the console, always black screen.

Making same process on VirtualBox, all is ok.
Making same process with GDM3, work well on Hyper-V.

I deduct the problem is probably on sddm side (as gdm3 handle well hyperv).

Some clues:
- https://forum.kaosx.us/d/2320-sddm-and-live-services-failing-to-start-when-using-windows-hyper-v
- https://github.com/systemd/systemd/issues/11299
- https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=931389

Thanks.

**Comment by jpwhiting** [comment]
Indeed, the same thing happens on a fresh Debian 10 install with KDE on hyperv. sddm is running, but no gui is shown. Switching to lightdm it shows the lightdm ui fine.

**Comment by Eifoen** [comment]
Same problem here but actually after switching VBox Backend to Windows-Virt-Platform. (Which I unfortunately needed to activate)

**Comment by BISMN** [comment]
Same problem

**Comment by Eifoen** [comment]
Actually this seems to be more of a VBox problem

after a lot of hassle and desperately wanting to use WSL2 => Hyper-V for docker integration I decided to give VMware Workstation a try. They support the Hyper-V backend for a few month now. Everything works like charm since I  switched.
Of course this involved some investment (sadly), but I entirely dropped VBox now...

**Comment by jpwhiting** [comment]
I hit it directly on hyper-v without virtualbox, just hyperv itself.

**Comment by hramrach** [comment]
See also systemd/systemd#20070
